test/test_suite/Experiments/Chinese/ModernChinese/README.md

```diff
@@ -1,56 +1,69 @@
 # Modern Chinese
 
-1. Sample Test
+Sample Test
 ---------------------------------------------------------------------
 In order to test if Lexos can analyze Modern Chinese literature, we 
 chose the "Turbulent Stream" trilogy and the "Love" trilogy written 
-by Ba jin, and two novels written by Jia Pingwa. Here we show how 
-cluster analysis can be used to identify the authorships, and to 
-show certain writer's writing style changes over the years.
+by Ba Jin, along with two novels written by Jia Pingwa. Here we show how 
+cluster analysis can be used to identify authorship, and to 
+show how a certain writer's writing style changes over the years.
 
 Using the tools from:   http://lexos.wheatoncollege.edu
 and the files found in the FilesToUse/SampleTest/ directory, you 
 should be able to produce a dendrogram as shown in ResultsToExpect/.
 
-###Sample Files:
+####Sample Files:
```

Comment:
Space needed between # and text

####Sample Files:

#### Sample Files:

Reply:
Thank you! I've added the spaces in.